docs/resources/cluster_v2.md

```diff
@@ -665,277 +837,287 @@ The following attributes are exported:
-* `worker_drain_options` - (Optional) Worker nodes drain options (list maxitems:1)
+* `control_plane_concurrency` - (Optional, string) How many control plane nodes should be upgraded at a time, 0 is infinite. Percentages are also accepted.
+* `control_plane_drain_options` - (Optional, list, max length: 1) Controlplane nodes drain options.
+* `worker_concurrency` - (Optional, string) How many worker nodes should be upgraded at a time. Percentages are also accepted.
+* `worker_drain_options` - (Optional, list, max length: 1) Worker nodes drain options.
 
-##### `control_plane_drain_options` `worker_drain_options`
+##### `control_plane_drain_options`, `worker_drain_options`
 
 ###### Arguments
 
-* `enabled` - (Optional) Drain options enabled? Default `true` (bool)
-* `force` - (Optional) Drain options force. Default `false` (bool)
-* `ignore_daemon_sets` - (Optional) Drain options ignore daemon sets. Default `true` (bool)
-* `ignore_errors` - (Optional) Drain options ignore errors. Default `false` (bool)
-* `delete_empty_dir_data` - (Optional) Drain options delete empty dir data. Default `false` (bool)
-* `disable_eviction` - (Optional) Drain options disable eviction. Default `false` (bool)
-* `grace_period` - (Optional/Computed) Drain options grace period (int)
-* `timeout` - (Optional/Computed) Drain options timeout (int)
-* `skip_wait_for_delete_timeout_seconds` - (Optional/Computed) Drain options skip wait for delete timeout seconds (int)
+* `enabled` - (Optional, bool, default: true) If `enabled` is set to true, nodes will be drained before upgrade.
+* `force` - (Optional, bool, default: false) If `force` is set to true, drain nodes even if there are standalone pods that are not managed by a ReplicationController, Job, or DaemonSet. Drain will not proceed without `force` set to true if there are such pods.
+* `ignore_daemon_sets` - (Optional, bool, default: true) If `ignore_daemon_sets` is set to false, drain will not proceed if there are DaemonSet-managed pods.
+* `ignore_errors` - (Optional, bool, default: false) If `ignore_errors` is set to true,  errors that occurred between drain nodes in group are ignored.
+* `delete_empty_dir_data` - (Optional, bool, default: false) if `delete_empty_dir_data` is set to true, continue draining even if there are pods using emptyDir (local storage).
+* `disable_eviction` - (Optional, bool, default: false) If `disable_eviction` is set to true, force drain to use delete rather than evict.
+* `grace_period` - (Optional/computed, int) Time in seconds given to each pod to terminate gracefully. If negative, the default value specified in the pod will be used.
+* `timeout` - (Optional/computed, int) Time to wait (in seconds) before giving up for one try.
+* `skip_wait_for_delete_timeout_seconds` - (Optional/computed, int) Skip waiting for the pods that have a DeletionTimeStamp > N seconds. Seconds must be greater than 0 to skip.
```

Comment:
It might be helpful to explicitly state that the pods will be force killed if they exceed this timeout value, but what's here also makes sense

Reply:
fixed.